helpdesk/helpdesk/settings.py

```diff
@@ -94,6 +94,7 @@
 INSTALLED_APPS = [
     "accounts",
     "display",
+    "helpdesk",
```

Comment:
:raised_eyebrow:

Reply:
I'd be interested to know which app you think the template tag belongs in instead?

Reply:
It was more that I was surprised this wasn't already here.